recipes-thunder/thunderclientlibraries/thunder-clientlibraries-4.3.0/Library-version-Matched-With-Release-ClientLibs.patch

```diff
@@ -0,0 +1,153 @@
+diff --git a/Source/bluetoothaudiosink/CMakeLists.txt b/Source/bluetoothaudiosink/CMakeLists.txt
+index f6dbd96..7498eda 100644
+--- a/Source/bluetoothaudiosink/CMakeLists.txt
++++ b/Source/bluetoothaudiosink/CMakeLists.txt
+@@ -21,7 +21,7 @@ cmake_minimum_required(VERSION 3.0)
+ find_package(WPEFramework)
+
+ project(bluetoothaudiosink)
+-project_version(1.0.0)
++project_version(4.1.0)
```

Comment:
shouldn't we have 4.3 here? (multiple places)

Reply:
Yes, indeed. IMO should be done automatically while tagging by Metrological. Comcast did patches ... IMO better to have that versioned correctly.
That patch was not applied at all, now is applied. Found also one more place - corrected.